<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-JKR11RL" failures="1" tests="6" name="Default test(failed)" time="36.084" errors="0" timestamp="2025-05-28T10:44:06 IST">
  <testcase classname="PracticeForListener.PracticeListener" name="CreateTest1" time="0.051">
    <skipped/>
  </testcase> <!-- CreateTest1 -->
  <testcase classname="PracticeForListener.PracticeListener" name="CreateTest1" time="0.027">
    <skipped/>
  </testcase> <!-- CreateTest1 -->
  <testcase classname="PracticeForListener.PracticeListener" name="CreateTest1" time="0.198">
    <skipped/>
  </testcase> <!-- CreateTest1 -->
  <testcase classname="PracticeForListener.PracticeListener" name="CreateTest1" time="0.025">
    <skipped/>
  </testcase> <!-- CreateTest1 -->
  <testcase classname="PracticeForListener.PracticeListener" name="CreateTest1" time="0.026">
    <skipped/>
  </testcase> <!-- CreateTest1 -->
  <testcase classname="PracticeForListener.PracticeListener" name="CreateTest1" time="0.032">
    <failure type="java.lang.AssertionError" message="expected [Home] but found [Admin123@ Administrator1 - Home - vtiger CRM 5 - Commercial Open Source CRM]">
      <![CDATA[java.lang.AssertionError: expected [Home] but found [Admin123@ Administrator1 - Home - vtiger CRM 5 - Commercial Open Source CRM]
at PracticeForListener.PracticeListener.CreateTest1(PracticeListener.java:15)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- CreateTest1 -->
  <testcase classname="PracticeForListener.PracticeListener" name="CreateTest2" time="0.0">
    <ignored/>
  </testcase> <!-- CreateTest2 -->
</testsuite> <!-- Default test(failed) -->
